rmsutil/system.py

```diff
@@ -0,0 +1,31 @@
+import contextlib
+import copy
+import os
+import sys
+
+
+@contextlib.contextmanager
+def push_env(**kwargs):
+    try:
+        old_env = os.environ.copy()
```

Comment:
I guess this is incidentally OK here, because you're never reassigning `os.environ`, but that should probably come with a warning in the code.